src/cmd/run/run.go

```diff
@@ -70,26 +74,53 @@ func run(ctx context.Context, names []string, options Options) error {
 		logging.Log("using image: %s", yeyContext.Image)
 	}
 
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "context:\n--\n%v--\n", yeyContext)
+	}
+
+	// Container name
 	containerName := yey.ContainerName(contexts.Path, yeyContext)
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "container: %s\n", containerName)
+	}
 
+	// Reset
 	if options.Reset {
+		if options.Verbose {
+			fmt.Fprintf(os.Stderr, "removing container first")
+		}
 		if err := docker.Remove(ctx, containerName); err != nil {
```

Comment:
i just realized there is a bug here. What if we run with reset but the container doesn't exist? We don't want to fail. Not to do with the PR but just thought I would flag it. I probably wrote the bug.